Equilobe.DailyReport.SL/ReportService.cs

```diff
@@ -76,16 +124,52 @@ private AtlassianUser ToAtlassianUser(JiraUser user, long instanceId)
             };
         }
 
-        private void UpdateDbUser(AtlassianUser dbUser, AtlassianUser updatedUser)
+        private List<AtlassianWorklog> GetWorklogsFromIssueWorklogs(List<JiraIssue> issueWorklog, List<AtlassianUser> users, long instanceId)
+        {
+            var worklogs = new List<AtlassianWorklog>();
+
+            foreach (var issue in issueWorklog)
+            {
+                foreach (var worklog in issue.fields.worklog.worklogs)
+                {
+                    var user = users.SingleOrDefault(p => p.Key == worklog.author.name);
+
+                    worklogs.Add(new AtlassianWorklog()
```

Comment:
remove the parentheses from AtlassianWorklog constructor. they are not mandatory if we use the object initializer, and it would look cleaner without them.